app/controllers/assignments_controller.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2012-2021, CVP Schweiz. This file is part of
+#  hitobito_cvp and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito.
+
+class AssignmentsController < CrudController
+  self.permitted_attrs = [:title, :description, :attachment_id, :person_id]
+
+  before_action :set_read_at, only: :show, if: :assignment_owner?
+
+  def create(options = {}, &block)
+    ActiveRecord::Base.transaction do
```

Comment:
can't u use super here?

or even use callbacks for notify ? https://github.com/codez/dry_crud#crud-controller-callbacks-